### ISSUE TITLE
M03-LAB06: Deploying docker containers to Azure App Service

### ISSUE THREAD

**Issue by asian-delirium** [MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions#530]
### Contact Details

ziyu1.wang@zeiss.com

### What should we change?

In the lab instruction for [Exercise 3 / Task 1 / Step 5](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/blob/master/Instructions/Labs/AZ400_M03_L06_Deploying_Docker_containers_to_Azure_App_Service_web_apps.md#task-1-add-a-new-role-assignment), 
the following `az` command is wrong / outdated: 

```bash
az role assignment create --assignee $spId --role $roleName --resource-group "rg-az400-container-NAME"
``` 
Specifically, `--resource-group` is not a recognized param and the required param `--scope` is missing, [see here](https://learn.microsoft.com/en-us/cli/azure/role/assignment?view=azure-cli-latest#az-role-assignment-create).

The correct command would look something like: 

```bash
az role assignment create --assignee $spId --role $roleName --scope "/subscriptions/<SUBSCRIPTION ID>/resourceGroups/rg-az400-container-<NAME>"
```

### Lab

Lab 06 Deploying Docker Containers to Azure App Service web apps

### Relevant screenshots

paste here 😉

**Comment by petender** [comment]
fixed with PR #532

**Comment by petender** [comment]
OK to close
